src/neural/network_mux.cc

```diff
@@ -153,7 +154,8 @@ class MuxingNetwork : public Network {
     }
   }
 
-  void Worker(Network* network, const int max_batch) {
+  void Worker(Network* network, const int max_batch, int id) {
+    Numa::BindThread(id);
```

Comment:
Logic for demux being + 1 probably kind of applies here as well?

Reply:
I'm still thinking about it.

Reply:
OK